test/util/conf_test.py

```diff
@@ -89,10 +93,19 @@ def test_flag_values(self):
     conf.load(flag_key='loaded_value')
     self.assertEquals('flag_value', conf.flag_key)
 
+  def test_non_str_flag_values(self):
+    self.assertEquals(True, conf.true_value)
+    self.assertEquals(100, conf.num_value)
+    # Make sure flag value takes precedence, even if a value is loaded.
+    conf.load(flag_key='loaded_value')
+    self.assertEquals(True, conf.true_value)
+
   def test_as_dict(self):
     conf.load(station_id='station_id')
     self.assertEquals({
         'flag_key': 'flag_value',
+        'true_value': True,
+        'num_value': 100,
         'cancel_timeout_s': 2,
         'enable_station_discovery': True,
```

Comment:
This key isn't showing up in conf._asdict() and I'm not sure why, and that makes this unit test fail. This fails on master too and I'm not sure what causes it; should I just take it out?

Reply:
looks like you fixed it, since travis is happy?